lib/symbioticpy/symbiotic/utils/utils.py

```diff
@@ -122,3 +123,13 @@ def get_symbiotic_dir():
     # get real path (strip off links)
     realpath = os.path.realpath(os.path.join(sys.argv[0], '..'))
     return os.path.abspath(os.path.dirname(realpath))
+
+def required_version(actual, required):
+    return LooseVersion(actual) >= LooseVersion(required)
+
+def get_clang_version():
```

Comment:
We should probably catch also the cases when the version is sth. like `3.8.0-2ubuntu4` or `5.0.0-3~16.04.1`. So we should split the string into two parts on `-` and get only the first part.

Reply:
This behavior should be already included in the `LooseVersion` comparison.